src/app/store/actions/ReferenceData.actions.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { Preparer } from '@app/models/preparer';
+import { TestStation } from '@app/models/test-station';
+import {
+  EReferenceDataActions,
+  LoadPreparers,
+  LoadPreparersSuccess,
+  LoadTestStations,
+  LoadTestStationsSuccess
+} from '@app/store/actions/ReferenceData.actions';
+
+describe('LoadPreparers', () => {
+  test('action should have correct type', () => {
+    const actionInstance = new LoadPreparers();
+    expect(actionInstance.type).toBe(EReferenceDataActions.LoadPreparers);
+  });
+});
+
+describe('LoadTestStations', () => {
+  test('action should have correct type & payload', () => {
+    const actionInstance = new LoadTestStations();
+    expect(actionInstance.type).toBe(EReferenceDataActions.LoadTestStations);
+  });
+});
+
+describe('LoadPreparersSuccess', () => {
+  test('action should have correct type & payload', () => {
```

Comment:
same comment as above also we are mentioning payload but doing no assertion of it.
Maybe the next assertion checks that the array/preparers is [] if we want to assert the payload/preparers. I think it is a good idea.

it('should have the correct action type and preparers paylaad when LoadPreparersSuccess() action is instantiated').